src/new-apps/components/AppListPage/utils.ts

```diff
@@ -5,11 +5,16 @@ import { InstalledApp } from "@saleor/new-apps/types";
 import { AppListPageSections } from "./types";
 
 export const resolveSectionsAvailability = ({
+  appsInstallations,
   installedApps,
   installableMarketplaceApps,
   comingSoonMarketplaceApps,
 }: AppListPageSections) => ({
-  installed: !installedApps || !!installedApps.length,
+  installed:
+    !installedApps ||
+    !!installedApps.length ||
+    !appsInstallations ||
+    !!appsInstallations.length,
```

Comment:
this contains a lot of conditional logic, should be tested

Reply:
Sure, I've added more tests